src/main/java/me/lokka30/microlib/other/Reflection.java

```diff
@@ -0,0 +1,54 @@
+package me.lokka30.microlib.other;
+
+import org.bukkit.Bukkit;
+import org.bukkit.entity.Player;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * Utility class to help with reflection & NMS.
+ *
+ * @author _ProfliX_
+ * @version 3.1.3
+ * @since 3.1.3
+ */
+public class Reflection {
+
+    /**
+     * Do not instantiate a utility class.
+     *
+     * @since 3.1.3
+     */
+    public Reflection() {
```

Comment:
why put a public constructor if it shouldn't be used

Reply:
Fixed.